plugin/bundles/weather_bit/resources/i18n/en.yml

```diff
@@ -0,0 +1,69 @@
+bundle_name: "Weatherbit"
+bundle_description: "Weather data and forecasts."
+credential_weather_bit_api_key_description: "API Key for authentication. Get it from https://www.weatherbit.io/account/dashboard"
+get_current_weather_input_lat_name: "Latitude"
+get_current_weather_input_lat_description: "The latitude of the location."
+get_current_weather_input_lon_name: "Longitude"
+get_current_weather_input_lon_description: "The longitude of the location."
+get_current_weather_input_units_name: "Units"
+get_current_weather_input_units_description: "The units of the temperature. M - [DEFAULT](Celsius), S - (Kelvin), I - (Fahrenheit)"
+get_current_weather_output_city_name_name: "City Name"
+get_current_weather_output_city_name_description: "The name of the city."
+get_current_weather_output_country_code_name: "Country Code"
+get_current_weather_output_country_code_description: "The country code of the city."
+get_current_weather_output_unit_name: "Unit"
+get_current_weather_output_unit_description: "The units of the temperature. M - [DEFAULT](Celsius), S - (Kelvin), I - (Fahrenheit)"
```

Comment:
Should be more specific and tell the model to only input a capital character, selecting from M, S, I